providers/google/tests/system/google/cloud/vertex_ai/example_vertex_ai_generative_model.py

```diff
@@ -25,28 +25,105 @@
 import os
 from datetime import datetime
 
+import requests
 from vertexai.generative_models import HarmBlockThreshold, HarmCategory, Part, Tool, grounding
 from vertexai.preview.evaluation import MetricPromptTemplateExamples
 
 from airflow.models.dag import DAG
 from airflow.providers.google.cloud.operators.vertex_ai.generative_model import (
     CountTokensOperator,
     CreateCachedContentOperator,
+    DeleteExperimentRunOperator,
     GenerateFromCachedContentOperator,
     GenerativeModelGenerateContentOperator,
     RunEvaluationOperator,
     TextEmbeddingModelGetEmbeddingsOperator,
 )
 
+
+def get_actual_models() -> dict[str, str]:
```

Comment:
LGTM
a small question: how many different places are we creating this method to list active models?

Reply:
I added this only for two system tests: this and  example_vertex_ai_generative_model_tuning.py

Reply:
I see. I was asking this because in case we have a lot of places we use this; maybe we can create a utils file to store this method in there and import it here and call it. But if it is not repeated that much, I think we can avoid doing that